[TEST-FAILURE] EntryProcessorStressTest.dropedEntryProcessorTest_withKeyOwningNodeTermination

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-problematicTest/com.hazelcast$hazelcast/225/testReport/com.hazelcast.map/EntryProcessorStressTest/dropedEntryProcessorTest_withKeyOwningNodeTermination/
```
java.lang.AssertionError: failed to execute all entry processor tasks at iteration
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.map.EntryProcessorStressTest$1.run(EntryProcessorStressTest.java:87)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:236)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:252)
	at com.hazelcast.map.EntryProcessorStressTest.dropedEntryProcessorTest_withKeyOwningNodeTermination(EntryProcessorStressTest.java:82)
```

--- a/hazelcast/src/test/java/com/hazelcast/map/EntryProcessorStressTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/map/EntryProcessorStressTest.java
@@ -28,7 +28,6 @@ import com.hazelcast.test.HazelcastSerialClassRunner;
 import com.hazelcast.test.HazelcastTestSupport;
 import com.hazelcast.test.TestHazelcastInstanceFactory;
 import com.hazelcast.test.annotation.NightlyTest;
-import com.hazelcast.test.annotation.ProblematicTest;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
@@ -42,13 +41,11 @@ import java.util.concurrent.locks.LockSupport;
 
 import static org.junit.Assert.assertTrue;
 
-
 @RunWith(HazelcastSerialClassRunner.class)
 @Category(NightlyTest.class)
 public class EntryProcessorStressTest extends HazelcastTestSupport {
 
     @Test
-    @Category(ProblematicTest.class)
     public void dropedEntryProcessorTest_withKeyOwningNodeTermination() throws ExecutionException, InterruptedException {
         String mapName = randomString();
         Config cfg = new Config();
@@ -72,7 +69,7 @@ public class EntryProcessorStressTest extends HazelcastTestSupport {
                 processorMap.submitToKey(key, new SimpleEntryProcessor(i));
 
                 if (i == maxTasks / 2) {
-                    instance2.getLifecycleService().terminate();
+                    instance2.getLifecycleService().shutdown();
                 }
             }
 

